index.html

```diff
@@ -75,108 +81,123 @@ <h1 class="primary-title font-family-poppins">I'm Belden.<br> Glad to see you!</
-            </ul>
-
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-            <ul class="column-links">
-               <li class="column-link"><a href="#">html</a></li>
-               <li class="column-link"><a href="#">css</a></li>
-               <li class="column-link"><a href="#">javascript</a></li>
-            </ul>
-
+            <div class="column-detail">
+                <h2 class="col-title">Facebook 360</h2>
+    
+                <ul class="canopy">
+                   <li class="canopy-title ">facebook</li>
+                   <li class="can-items">
+                     <span>• </span>  Full Stack Dev <span>• </span>
+                   </li>
+                   <li class="can-items">2015</li>
+                </ul>
+    
+                <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+    
+                <ul class="column-links">
+                   <li class="column-link"><a href="#">html</a></li>
+                   <li class="column-link"><a href="#">ruby on rails</a></li>
+                   <li class="column-link"><a href="#">css</a></li>
+                   <li class="column-link"><a href="#">javascript</a></li>
+                </ul>
+    
+    
+                <a href="" class="column-button"> see project</a>
 
-            <a href="" class="column-button"> see project</a>
+            </div>
          </div>
 
-         <div class="col-1 column-item">
+         <div class="col-1 column-item row-reverse">
             <img src="images\Snapshoot (1).jpg" alt="">
-            <h2 class="col-title">Multi-post <br> stories</h2>
+            <div class="column-detail">
+                
+            <h2 class="col-title">Uber Navigation</h2>
 
             <ul class="canopy">
-               <li class="canopy-title ">canopy</li>
+               <li class="canopy-title ">Uber</li>
                <li class="can-items">
-                 <span>• </span>  backend-dev <span>• </span>
+                 <span>• </span>  Lead Developer <span>• </span>
                </li>
-               <li class="can-items">2015</li>
+               <li class="can-items">2018</li>
             </ul>
 
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
 
             <ul class="column-links">
                <li class="column-link"><a href="#">html</a></li>
+               <li class="column-link"><a href="#">ruby on rails</a></li>
                <li class="column-link"><a href="#">css</a></li>
                <li class="column-link"><a href="#">javascript</a></li>
             </ul>
 
 
             <a href="" class="column-button">see project</a>
+            </div>
```

Comment:
I must say you just nailed it styling the project cards 👏 Amazing. Just one more thing to go 
- [ ] I think it would be better if you follow the figma designs and add colors that are given in figma template in order to follow the project requirements. While you have done an awesome job personalizing the cards and adding `background-color: #fffafa`, but let's make sure that we follow the project requirements. In figma the background color for cards is `#fff`
## **_Your design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214614847-d829200b-15db-4645-81d2-83e85554cd82.png)

## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214612945-3fdfae47-abeb-4d86-b943-a99ee9d5e05a.png)